src/main/resources/org/broadinstitute/hellbender/utils/plotter/TangentResultsPlotting.R

```diff
@@ -94,98 +134,55 @@ QC = function(dat) {
 }
 
 ## Chromosome Background for data
-SetUpPlot = function(y.lab, y.min, y.max, x.lab, lab.chr, sex_chrs){
-  ## Via http://genome.ucsc.edu/cgi-bin/hgTables
-  ## HG19 chromosome lengths
-  chr.lens = c(249250621, 243199373, 198022430, 191154276, 180915260, 171115067, 159138663,
-        146364022, 141213431, 135534747, 135006516, 133851895, 115169878, 107349540,
-        102531392, 90354753, 81195210, 78077248, 59128983, 63025520, 48129895, 51304566)
-
-  ## HG19 centromere positions left and right bounds
-  cent.posS = c(121535434, 92326171, 90504854, 49660117, 46405641, 58830166, 58054331, 43838887, 47367679,
-	39254935, 51644205, 34856694, 16000000, 16000000, 17000000, 35335801, 22263006, 15460898, 24681782,
-	26369569, 11288129, 13000000)
-  cent.posE = c(124535434, 95326171, 93504854, 52660117, 49405641, 61830166, 61054331, 46838887, 50367679,
-	42254935, 54644205, 37856694, 19000000, 19000000, 20000000, 38335801, 25263006, 18460898, 27681782,
-	29369569, 14288129, 16000000)
-
-  ###Add sex chromosomes###
-  if(sex_chrs){
-      ## Add X
-      chr.lens = c(chr.lens, 155270560)
-      cent.posS = c(cent.posS, 58632012)
-      cent.posE = c(cent.posE, 61632012)
-
-      ## Add Y
-      chr.lens = c(chr.lens, 59373566)
-      cent.posS = c(cent.posS, 10104553)
-      cent.posE = c(cent.posE, 13104553)
-  }
-
-  genome.len = sum(chr.lens)
-  chr.w = chr.lens / genome.len
+SetUpPlot = function(y.lab, y.min, y.max, x.lab, label_chromosomes, sex_chrs){
   suppressWarnings( par( mar=c(3.1,3.6,0.1,0),mgp=c(2,-0.2,-1.1) ) )
   plot(0, type = "n", bty = "n", xlim = c(0, 1), ylim = c(y.min, y.max),
        xlab = "", ylab = "", main = "", xaxt = "n")
   mtext(side=1, line=1.5, x.lab, cex=0.75, outer=FALSE)
   mtext(side=2.2, line=1.5, y.lab, cex=0.75, las=FALSE, outer=FALSE)
-  chr.mids=cumsum(chr.w)-chr.w/2
-  if(lab.chr){
-  	lab.vals = (c(1:length(chr.w)))
-  	odd.ix = lab.vals %% 2 == 1
-  	if(sex_chrs){
```

Comment:
Why not use this for plotting? I think people expect to see X/Y instead of 23/24